examples/grayscale_rgba_example.html

```diff
@@ -0,0 +1,47 @@
+<html>
+
+<body>
+    <img id="img" src="pinball.jpg">
+    <canvas id="canvas" width="1637" height="2048"></canvas>
+    <canvas id="dst" width="1637" height="2048"></canvas>
+
+    <script type="module">
+        import jsfeatCpp from "./../build/jsfeatES6cpp_debug.js"
+        const jsfeat = await jsfeatCpp();
+        console.log(jsfeat);
+
+        var canvas = document.getElementById('canvas');
+        var ctx = canvas.getContext('2d');
+        var canvasDst = document.getElementById('dst');
+        var ctxDst = canvasDst.getContext('2d');
+        var img = document.getElementById('img');
+        ctx.drawImage(img, 0, 0);
+        var data = ctx.getImageData(0, 0, img.width, img.height);
+
+        const RGBA2GRAY = jsfeat.Colors.COLOR_RGBA2GRAY.value;
+        const U8_t = jsfeat.Types.U8_t.value;
+        const C1_t = jsfeat.Types.C1_t.value;
+
+        var m_u8 = new jsfeat.Matrix_smart(img.width, img.height, U8_t | C1_t);
+        console.log(m_u8);
+        var imgproc = new jsfeat.imgproc();
+
+        if (data) {
+            imgproc.grayscale_rgba(data.data, img.width, img.height, m_u8.getSmartPointer());
+            
+            const imageData = ctxDst.createImageData(img.width, img.height);
+            var inputData = m_u8.data;
+            m_u8.delete()
```

Comment:
We should delete the instance to avoid memory leaks, embind warn for this, but even deleting i receive the message:
```
jsfeatES6cpp_debug.js:2030 Embind found a leaked C++ instance Matrix_smart <0x507e88>.
We'll free it automatically in this case, but this functionality is not reliable across various environments.
Make sure to invoke .delete() manually once you're done with the instance instead.
Originally allocated
    at Matrix_smart.Matrix_smart$getSmartPointer [as getSmartPointer] (eval at new_ (http://localhost:5500/build/jsfeatES6cpp_debug.js:2617:27), <anonymous>:9:19)
    at http://localhost:5500/examples/grayscale_rgba_example.html:30:75
(anonymous) @ jsfeatES6cpp_debug.js:2030
```
Probabòy it's needed a custom deleter?